.semaphore/cp_dockerfile_build.yml

```diff
@@ -25,6 +25,7 @@ global_job_config:
     commands:
       - checkout
       - sem-version java 8
+      - sem-version python unknown
```

Comment:
not sure if this is correct

Reply:
lets keep it same as master, i'll make this change

Reply:
@jainruchir could you remove this line.. we can add it later.. for promotion, we don't need this anyway

Reply:
removed